src/util/reference_counting.h

```diff
@@ -121,9 +123,9 @@ void reference_counting<T>::remove_ref(dt *old_d)
   if(old_d==nullptr)
     return;
 
-  assert(old_d->ref_count!=0);
+  PRECONDITION(old_d->ref_count != 0);
 
-  #ifdef REFERENCE_COUNTING_DEBUG
```

Comment:
This seems like an odd change, was this done by clang-format?

Reply:
I have undone those indentations and it did not happen again upon a clang-format. (!?!)